src/interpreter.rs

```diff
@@ -1081,7 +1101,8 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
                         Err(e) => return Err(e),
                     }
                 } else {
-                    panic!("unimplemented: primitive read of fat pointer type: {:?}", ty);
+                    panic!("unimplemented: primitive read of fat pointer type: {:?}",
+                           ty);
```

Comment:
Er, I'm actually a bit confused about this. I use `rust.vim` which sets the `textwidth` to 99 which I considered the community norm (well, something around 100, anyway) and this line was only 90 chars wide before. This change doesn't bring it below 80, either.

If it can fit within 100 chars, I wouldn't generally change it.